### PR TITLE
Update links in readme after repository moved to cardano-foundation

### PR DESCRIPTION
**Pull request by Anviking**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,21 +77,21 @@ See **Installation Instructions** for each available [release](https://github.co
...
-| [Cardano Wallet Documentation](https://input-output-hk.github.io/cardano-wallet/)                                     | Anyone interested in the project and our development process |
-| [Adrestia Documentation](https://input-output-hk.github.io/adrestia/)                                     | Anyone interested in the project and our development process |
+| [User Guide](https://cardano-foundation.github.io/cardano-wallet/user-guide) | Users of Cardano Wallet                              |
+| [CLI Manual](https://cardano-foundation.github.io/cardano-wallet/user-guide/cli) | Users of the Cardano Wallet API                              |
+| [API Documentation](https://cardano-foundation.github.io/cardano-wallet/api/edge)                     | Users of the Cardano Wallet API                              |
+| [Cardano Wallet Documentation](https://cardano-foundation.github.io/cardano-wallet/)                                     | Anyone interested in the project and our development process |
+| [Adrestia Documentation](https://cardano-foundation.github.io/adrestia/)                                     | Anyone interested in the project and our development process |
```
Comment (Anviking) [inline README.md]: Should iohk, still

Comment (Anviking) [reply README.md]: Fixed.

**Review by paweljakubas — APPROVED** [review]
:+1:

**Review by 19714 — APPROVED** [review]

**Pull request merged**